sql-to-dbsp-compiler/SQL-compiler/pom.xml

```diff
@@ -231,5 +231,12 @@
             <artifactId>jcommander</artifactId>
             <version>1.82</version>
         </dependency>
+        <!-- HyperSQL Java database -->
```

Comment:
Shouldn't this only be a test dependency?

Reply:
Initially it was, but there is a class which wraps a HSQLDB server which is not part of the test code.